### ISSUE TITLE
Unable to downgrade from 1.13.0 to 1.12.2 in Windows 10 / Access is denied. (os error 5)

### ISSUE THREAD

**Issue by ngdangtu-vn** [denoland/deno#11703]
Because a bug was introduced in http_errors when I updated Deno to 1.13.0, I have to downgrade it to lower version so I can keep using Opine.

However this is what I got when I attempted to downgrade it.

```
C:\xyz>deno upgrade --version 1.12.2
Checking https://github.com/denoland/deno/releases/download/v1.12.2/deno-x86_64-pc-windows-msvc.zip
24.4 MiB / 24.4 MiB (100.0%)
Deno is upgrading to version 1.12.2
error: Access is denied. (os error 5)
```

I also made sure to google the problem and found one issue #5301 

From there I tried all the terminal I have. I also tried the second solution in the issue above. But nothing works.

![image](https://user-images.githubusercontent.com/68631458/129444752-82e32c5f-2e9a-411f-bffd-5e913ad59c5f.png)

Here is my spec
```
deno 1.13.0 (release, x86_64-pc-windows-msvc)
v8 9.3.345.11
typescript 4.3.5

Windows 10 20H2 19042.1165
```

Before that I upgrade to Deno 1.13.0 via the same command: `deno upgrade`

**Comment by dsherret** [comment]
Bit of a guess, but perhaps the error is caused by the deno process running? (check `Get-Process deno` and stop them if able `Get-Process deno | Stop-Process`) Maybe try closing your IDE before upgrading/downgrading.

**Comment by ngdangtu-vn** [comment]
> Bit of a guess, but perhaps the error is caused by the deno process running? ... 
That doesn't make sense because I was able to upgrade to 1.13.0 without any trouble. I remember that I ran deno a few time before and I only started upgrade deno when a dep remind me to get the latest version of Deno. However...

> Maybe try closing your IDE before upgrading/downgrading.
I think your guess on IDE could be possible. Both the first time encountering error and the second after restarting machine, I all started vscode with a deno plugin activated workspace immediately after that and got the `os error 5` for trying to upgrade. Fortunately, today when I got your message and try it again while I made sure no Deno process is running then it works well.


Since it works for me now I'll close the issue. But I hope we could have a tip message for this common error in the future.
